src/utils/breakpoints.js

```diff
@@ -1,20 +1,20 @@
 import { useMediaQuery } from "react-responsive";
 
 const Desktop = ({ children }) => {
-  const isDesktop = useMediaQuery({ minWidth: 768 });
+  const isDesktop = useMediaQuery({ minWidth: 1024 });
   return isDesktop ? children : null;
 };
-// const Tablet = ({ children }) => {
-//   const isTablet = useMediaQuery({ minWidth: 768, maxWidth: 991 });
-//   return isTablet ? children : null;
-// };
+const Tablet = ({ children }) => {
+  const isTablet = useMediaQuery({ minWidth: 640, maxWidth: 1024 });
+  return isTablet ? children : null;
+};
 const Mobile = ({ children }) => {
-  const isMobile = useMediaQuery({ maxWidth: 767 });
+  const isMobile = useMediaQuery({ maxWidth: 640 });
```

Comment:
this should be constant

Reply:
We've both `const Mobile` and `const isMobile`.
I didn't get it.

Reply:
I guess what he meant was, Make values like 640 and 1024 constants and use the constant variables instead of the values directly so that if we want to change the values in future then we wont have to make changes in a hundred places.

Reply:
@ksraj123, I've made the needed changes.